rfcs/proposed/range_algorithms/README.md

```diff
@@ -0,0 +1,36 @@
+# Support the second portion of the oneDPL Range APIs
+
+## Introduction
+Based on statistics (observing C++ code within github.com) for the usage of popular algorithms, the following
+range-based APIs are suggested to be implemented next in oneDPL.
+`fill`, `generate`, `move`, `replace`, `replace_if`, `remove`, `remove_if`, `mismatch`, `minmax_element`, `minmax`,
+`min`, `max`, `find_first_of`, `find_end`, `is_sorted_until`
+
+## Motivations
+The feature is proposed as the next step of range-based API support for oneDPL.
+
+### Key Requirements
+- The range-based signatures for the mentioned API should correspond to the [proposed specification](https://github.com/uxlfoundation/oneAPI-spec/pull/614)
+that is based on the [C++ standardization proposal P3179](https://wg21.link/p3179).
+- The proposed implementation should support all oneDPL execution policies: `seq`, `unseq`, `par`, `par_unseq`, and `device_policy`.
+
+### Implementation proposal
+- The implementation is supposed to rely on existing range-based or iterator-based algorithm patterns, which are already
+implemented in oneDPL.
+- Several algorithms described in P3179 have slightly different semantics. To implement these, some existing algorithm patterns
+might require modifications or new versions.
```

Comment:
I don't think we need to go into the details of each, I think we should at least list the APIs with semantic differences which may need modifications

Reply:
probably, it is reasonable..